tools/extras/install_irstlm.sh

```diff
@@ -55,6 +55,7 @@ fi
   [ ! -z ${IRSTLM} ] && \
     echo >&2 "IRSTLM config is already in env.sh" && exit
 
+  wd=`pwd`
   wd=`readlink -f $wd 2>/dev/null || pwd`
```

Comment:
I'm OK with wd=`pwd -P`; it works on my mac.


On Wed, May 4, 2016 at 8:07 PM, Giulio Paci <notifications@github.com>
wrote:

> In tools/extras/install_irstlm.sh
> <https://github.com/kaldi-asr/kaldi/pull/759#discussion_r62134742>:
>
> > @@ -55,6 +55,7 @@ fi
> >    [ ! -z ${IRSTLM} ] && \
> >      echo >&2 "IRSTLM config is already in env.sh" && exit
> >
> > +  wd=`pwd`
> >    wd=`readlink -f $wd 2>/dev/null || pwd`
>
> Reading http://www.unix.com/man-page/posix/1posix/pwd/ probably the
> desired behaviour is just one line:
>
> wd=`pwd -P`
>
> or, for maximum compatibility with already used systems:
>
> wd=`pwd -P 2>/dev/null || pwd`
>
> This will avoid the unportable readlink -f, provide a portable solution
> and will be equivalent to or better than (from the performance point of
> view, that is completely irrelevant in this case) readlink -f $PWD in
> bash where pwd is a builtin.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/759/files/3494152412cb39eb55b7104ef0b72e4192eb3bb2#r62134742>
>